src/client/view/mostUsedWords.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { SlidingSearchBar } from './searchBar';
+
+const WordInfo = ({ wordInfo }) => (
+  <tr>
+    <td className='right aligned column'>{wordInfo.count}</td>
+    <td className='left aligned column'>{wordInfo.word}</td>
+    <td className='right aligned column'>
+      <div className='ui checkbox'>
+        <input type='checkbox'  defaultChecked='true'/>
+      </div>
+    </td>
+  </tr>
+);
+
+const Words = React.createClass({
+  componentDidMount() {
+    $('.ui.checkbox').checkbox();
+  },
+
+  componentDidUpdate() {
+    $('.ui.checkbox').checkbox();
+  },
+
+  render() {
+    return (
+      <div style={{ 'height': '300px', 'overflow-y': 'scroll' }}>

```

Comment:
????

a) it's an object, so you quote the keys
b) since when has overflowY been a thing? (It works, but why would I use it vs the standard overflow-y?)